src/arc_utilities/color_mapping.py

```diff
@@ -93,19 +93,23 @@ def rad_diff(x, y):
         return diff
 
 
-def adjust_hue((m_saturated, s_saturated, h_saturated), m_unsaturated):
```

Comment:
didn't know tuple parameter unpacking was a thing in python 2...

Reply:
neither did I :upside_down_face: